### PR TITLE
error-reporting: Fix README typos showing usage

### PR DESCRIPTION
**Pull request by DominicKramer**

### CODE REVIEW

File: packages/error-reporting/README.md [hunk 1]
```diff
@@ -199,26 +199,26 @@ server.route({
...
+server.register(errors.hapi);
 ```
 
 ### Using Koa
 
 ```js
 var errors = require('@google-cloud/error-reporting')();
 var koa = require('koa');
-var app = koa();
+var app = new koa();
```
Comment (stephenplusplus) [inline packages/error-reporting/README.md]: Let's require `koa` under the alias `Koa`, so conventionally, the name is a constructor.

Comment (DominicKramer) [inline packages/error-reporting/README.md]: Good catch.  I've updated it.

**Comment by stephenplusplus** [comment]
LGTM -- do you want to ping anyone else for a review?

**Comment by DominicKramer** [comment]
@stephenplusplus I've sent another review out.

File: packages/error-reporting/README.md [hunk 2]
```diff
@@ -199,26 +199,26 @@ server.route({
...
+server.register(errors.hapi);
 ```
 
 ### Using Koa
 
 ```js
 var errors = require('@google-cloud/error-reporting')();
-var koa = require('koa');
-var app = koa();
+var Koa = require('koa');
```
Comment (kjin) [inline packages/error-reporting/README.md]: For the sake of consistency, could you move this line above the `errors` one?

Comment (DominicKramer) [inline packages/error-reporting/README.md]: Fixed

**Comment by DominicKramer** [comment]
@stephenplusplus The test failures look unrelated to this PR.  Other than the test failures, this PR is ready to land.

**Comment by stephenplusplus** [comment]
Could you paste the errors from Circle? Mobile and Circle log outputs do not get along.

**Comment by DominicKramer** [comment]
@stephenplusplus Below are sections of the test output that I think are relevant.
```
. scripts/circle/test.sh
...
> google-cloud-node@ postinstall /home/ubuntu/google-cloud-node
> node ./scripts/install.js
...
Installing dependencies for dns
Installing dependencies for error-reporting
Installing dependencies for google-cloud
Installing dependencies for language
...
> google-cloud-node@ lint /home/ubuntu/google-cloud-node
> jshint scripts/ packages/ system-test/ test/ && jscs packages/ system-test/ test/

git remote add temp https://github.com/GoogleCloudPlatform/google-cloud-node.git
git fetch -q temp
Warning: Permanently added the RSA host key for IP address '192.30.253.113' to the list of known hosts.

git diff HEAD temp/master --name-only
.jscsrc
.jshintignore
docs/manifest.json
packages/error-reporting/README.md
packages/error-reporting/doc/images/errors-overview.png
packages/error-reporting/src/interfaces/hapi.js
packages/error-reporting/test/unit/interfaces/hapi.js
packages/logging/package.json
packages/logging/src/log.js
packages/logging/src/metadata.js
packages/logging/system-test/logging.js
packages/logging/test/log.js
packages/logging/test/metadata.js
packages/video-intelligence/package.json
packages/video-intelligence/src/index.js
packages/video-intelligence/src/v1beta1/index.js
packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js
packages/video-intelligence/src/v1beta1/video_intelligence_service_client_config.json
packages/video-intelligence/test/gapic-v1beta1.js
scripts/docs/config.js
scripts/helpers.js

module.js:327
    throw err;
    ^

Error: Cannot find module '/home/ubuntu/google-cloud-node/packages/video-intelligence/package.json'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at new Module (/home/ubuntu/google-cloud-node/scripts/helpers.js:63:17)
    at Module (/home/ubuntu/google-cloud-node/scripts/helpers.js:57:12)
    at Array.map (native)
    at Function.Module.getUpdated (/home/ubuntu/google-cloud-node/scripts/helpers.js:142:24)
    at Object.<anonymous> (/home/ubuntu/google-cloud-node/scripts/build.js:50:22)
    at Module._compile (module.js:409:26)

. scripts/circle/test.sh returned exit code 1
```

**Comment by stephenplusplus** [comment]
Thanks!

**Pull request merged**